src/main/java/com/GeorgesServer/app/com/GeorgesServer/response/IResponseBuilder.java

```diff
@@ -0,0 +1,10 @@
+package com.GeorgesServer.app.com.GeorgesServer.response;
+
+public interface IResponseBuilder {
```

Comment:
I foresee this growing to eventually have methods such as setBody(), setHeaders().